libs/design/tslint.json

```diff
@@ -0,0 +1,17 @@
+{
+    "extends": "../../tslint.json",
+    "rules": {
+        "directive-selector": [
+            true,
+            "attribute",
+            "lib",
+            "camelCase"
+        ],
+        "component-selector": [
+            true,
+            "element",
+            "lib",
+            "kebab-case"
+        ]
+    }
+}
```

Comment:
indentation should be 2 spaces instead of 4

Reply:
@lderrickable this is something than we can enforce in CI. `ng lint`. Will fix in another PR.

Reply:
@lderrickable I've opened #106 to cover this.